README.md

```diff
@@ -59,6 +59,8 @@ workdir/vmlinux.container
 
 
 ```
+#Pull and setup latest kernel for Clear Containers
+sudo -E make kernel-src
```

Comment:
I would say that this needs a `$ ` at the start. But your change is consistent with the rest of the document so I think adding a prompt can be handled on a separate PR.

Reply:
Sure, added https://github.com/clearcontainers/osbuilder/issues/20